README.md

```diff
@@ -25,53 +25,41 @@ To learn how to use conda-store-ui alongside conda-store, please visit [the cond
 
 ## Development
 
-### Setting up the development environment
+To get started with conda-store-ui development, there are a couple of options, depending on the type of changes you are working on.
+This guide will help you to set up your local development environment.
 
-We use [Docker Compose](https://docs.docker.com/compose/) to set up the infrastructure and recommend using `conda` as the package manager for NodeJS and yarn.
+### Prerequsites
+Before setting up conda-store ui, you must prepare your environment. 
 
-You will need to have the following installed:
+We use [Docker Compose](https://docs.docker.com/compose/) to set up the infrastructure before starting ensure that you have docker-compose installed. If you need to install docker-compose, please see their [installlation documentation](https://docs.docker.com/compose/install/)
 
-- [Docker compose](https://docs.docker.com/compose/install/)
-- conda or mamba
+1. Clone the [conda-store-ui](https://github.com/conda-incubator/conda-store-ui.git) repository.
+2. Copy `.env.example` to `.env`. All default settings should work, but if you want to test against a different version of conda-store-server, you can specify if in the `.env` file by setting the `CONDA_STORE_SERVER_VERSION` variable to the desired version. Refer to the [Configuration documentation](https://conda-incubator.github.io/conda-store-ui/?path=/docs/docs-configuration--page) for more information on the `.env` file.
 
-To get started:
+### Local Development with conda-store-ui running in Docker
+Running conda-store-ui in Docker is the most straightforward way to set up your local development environment.
 
-1. Clone this repo and from the root of the directory, start Docker compose:
+1. Run `yarn run start:docker` to start the entire development stack.
```

Comment:
We need to run `yarn install` first

Reply:
No, you don't. That is taken care of by docker. The docker container does that when it is built.